server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    sitter_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    start_date: {
+      type: Date,
+      required: true,
+    },
+    end_date: {
+      type: Date,
+      required: true,
+    },
+    accepted: {
+      type: Boolean,
+      default: false,
+    },
+    declined: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
These two fields could definitely do the job but more often than not, the logic for keeping these in a valid state (e.g. both `accepted` and `declined` aren't `true` at the same time) become awkward real quick. So I recommend reducing these to a single field (e.g. `status`) with an [enum validator](https://mongoosejs.com/docs/validation.html#built-in-validators).

Reply:
Added status with enum validator.